pyproject.toml

```diff
@@ -130,6 +130,7 @@ odes = [
 all = [
     "autograd>=1.6.2",
     "scikit-fem>=8.1.0",
+    "parameterized>=0.9",
```

Comment:
I would suggest moving this to the `[dev]` extra since the only import of it at this time is in the unit tests (unless it's going to be used for user-facing functionality too in the future, in which case it can stay here, that is)

Reply:
I don't think this works, since `nox -s unit` fails (`No module named parameterized`); taking a deeper look, the noxfile installs `[all]` for tests (and selected others), but none of these include `[dev]` dependencies. Do we want to add `dev` to tests, or create a new 'tests' environment (and should that be part of this PR)?

Reply:
Yes, that makes sense. We have been using the `[dev]` extra to collect a set of test-related dependencies; we should add them for the `nox` environments for the unit, integration, and coverage tests (should be fine to include in this PR since it is a minimal change at best)